packages/base/src/types/Behaviour.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Behaviour => Behavior (we use American spelling)
Should also be ItemNavigationBehavior, because it is very specific and nothing else will use it